assignment/tool/src/webapp/vm/assignment/assignment_macros.vm

```diff
@@ -384,4 +384,70 @@
 		</div>
 	#end
 #end
+
+## macro to display content review results; assumes they're visible (caller must do the required checks / provide appropriate messaging)
+#macro(contentReviewReports $submission $inTable $displayUrkundOptOut)
+	#set($reviewResults=$!service.getContentReviewResults($!submission))
+	#if ($reviewResults.size() >= 3)
+		#if (!$inTable)
+			<div class="contentReviewDisclosure discTria">
+		#else
+			<div class="contentReviewDisclosure">
+		#end
+			<a href="#" class="contentReviewDisclosure" onclick="ASN.handleReportsTriangleDisclosure(this, this.children[0], this.parentNode.parentNode.children[1], '$tlang.getString('review.report.expand')', '$tlang.getString('review.report.collapse')'); return false;" aria-label="$tlang.getString('review.report.expand')" aria-expanded="false" aria-controls="reportsDiv_$submission.id" >
+				<img class="disclosureTriangle" src="#imageLink("sakai/expand.gif")" />
```

Comment:
Reviewers:
The img tag is nothing more than a "visual" queue, and is contained inside an anchor with aria-labels. So I  think we're good; if you agree, please "Resolve conversation", or let me know otherwise! Thanks!

Reply:
> Images with only visual value should have an empty alt attribute set on them.  Omitting the alt attribute makes most screen readers read out the entire image URL and providing an alt attribute when the image is for visual purposes only is just as useless.

https://davidwalsh.name/accessibility-tip-empty-alt-attributes